package.json

```diff
@@ -126,7 +126,7 @@
     "html-loader": "0.5.5",
     "html-to-react": "1.3.4",
     "html-webpack-plugin": "3.2.0",
-    "i18n-calypso": "3.0.0",
+    "i18n-calypso": "file:./packages/i18n-calypso",
```

Comment:
Highlighting that this _will_ put the package into production

Reply:
Yes, this is the best part 😈